.gitignore

```diff
@@ -83,6 +83,7 @@ static/usr/share/man/man8/vdsmd.8
 vdsm/rpc/vdsm-api.html
 vdsm_hooks/openstacknet/sudoers
 vdsm_hooks/checkips/vdsm-checkips.service
+.vagrant
```

Comment:
Hidden files appear at the beginning of the list. I would expect this right after `.tox`.